templates/.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
It is the first that I see this file. Personally, I slightly prefer overriding the rules in the `.eslintrc.json` file, just to avoid to "pollute" the root directory with another "*rc* file. 🤔 
But it is just a matter of taste :)

Reply:
Well… 😁 it's not only a matter of taste. I totally see your point of not wanting a whole lot of files in the project root. An advantage of using these separate tool-specific files is that you can use them on their own (they'll usually be picked up by the individual tool as well as the integration for ESLint in this case). Just a thought 🤔